config/locales/en.yml

```diff
@@ -315,7 +315,7 @@ en:
             label: "No"
         custom_select_error: "Select if you can offer transport or logistics"
       transport_type:
-        title: "What kind of services can you offer?"
+        title: "What kind of transport or logisitcs services can you offer?"
```

Comment:
Typo: should be "logistics"

Reply:
"logistics services" sounds a bit odd to my ears. What about "logistical services" instead?

Reply:
Thanks for spotting that typo! 
Google search trends suggests 'logistics services' is more popular than 'logistical services', so let's stick with that.